src/ai/backend/manager/models/image.py

```diff
@@ -84,14 +84,14 @@
 )
 
 
-class PublicImageLoadFilter(enum.StrEnum):
+class PublicImageLoadType(enum.StrEnum):
     OPERATIONAL = "operational"
     """Include operational images."""
     CUSTOMIZED = "customized"
     """Include customized images owned or accessible by API callee."""
 
 
-class ImageLoadFilter(enum.StrEnum):
+class ImageLoadType(enum.StrEnum):
```

Comment:
Why not just `ImageType`?

Reply:
`ImageType` is already being used [here](https://github.com/lablup/backend.ai/blob/4446f594d654febde7c2481cc3bbb653b7d59385/src/ai/backend/manager/models/image.py#L167-L171).